src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/common/pooling.cpp

```diff
@@ -154,6 +154,22 @@ INSTANTIATE_TEST_SUITE_P(smoke_MaxPoolV8_CPU_5D_ref, MaxPoolingV8LayerCPUTest,
                                  ::testing::Values(expectedCpuConfig())),
                          MaxPoolingV8LayerCPUTest::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(smoke_MaxPoolV14_CPU_5D, MaxPoolingV14LayerCPUTest,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(paramsMaxV85D()),
+                                 ::testing::ValuesIn(inputShapes5D()),
+                                 ::testing::ValuesIn((inpOutPrecision())),
+                                 ::testing::ValuesIn(vecCpuConfigs)),
+                         MaxPoolingV14LayerCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_MaxPoolV14_CPU_5D_ref, MaxPoolingV14LayerCPUTest,
```

Comment:
Are there any single layer tests for new version of pool operators using new ceiling mode?

Reply:
It's something I asked about in the PR description - `AvgPool` layer tests seems to be using `ov::test::PoolingLayerCPUTest` instead of `AvgPool`-specific class like in case of `MaxPool` - could anyone confirm how I should go about adding proper `AvgPool` tests?

Reply:
@mvafin || @rkazants || @mitruska could you help with Przemek's question here?

Reply:
I figured it out, I'm currently working on adding `AvgPool` tests. However, the `MaxPool-14` tests are failing with `Couldn't find input with name Parameter_XXXXX in the inputs map` or `Batch size is zero.` when using the exact same test cases as `MaxPool-8`- if that's caused by a common mistake when adding new layer tests classes please let me know.

Reply:
We had a sync with @p-wysocki, checked potential root causes and some minor issues, but the problem mentioned above is not resolved yet. `AvgPool` tests are in progress.

Reply:
The cause may be in the downgrade transformation. If so, please fix it before disabling.

Reply:
I found no issues in the downgrade transformation. What's surprising, tests for `MaxPool-8` are passing, but `MaxPool-14`, `AvgPool-1` and `AvgPool-14` tests are failing with the mentioned errors. The cause is not operator specific.